src/supa/job/reserve.py

```diff
@@ -358,7 +369,7 @@ def __init__(self, connection_id: UUID):
         Args:
            connection_id: The connection_id of the reservation commit request
         """
-        self.log = logger.bind(job=self.__class__.__name__)
+        self.log = logger.bind(job=self.__class__.__name__, connection_id=str(connection_id))
```

Comment:
No need to cast connection_id whilst logging

Reply:
Because connection_id is an UUID it will show up in the logs like this:

```connection_id=UUID('76cf1a86-4439-420d-8089-27d45f9d77af')```

It is preferred to log it like a string:

```connection_id=76cf1a86-4439-420d-8089-27d45f9d77af```